src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -248,8 +249,11 @@ def _load_version(cls, state, version):
         assert(version == cls._PYTHON_ONE_SHOT_OBJECT_DETECTOR_VERSION)
         # we need to undo what we did at save and turn the proxy object
         # back into a Python class
-        state['detector'] = _ObjectDetector._load_version(
-            state['detector'], state["_detector_version"])
+        if USE_CPP:
+            state['detector'] = _ObjectDetector_beta(state['detector'])
+        else:
+            state['detector'] = _ObjectDetector._load_version(
```

Comment:
Let’s not depend on the old OD implementation here, since we want to delete it. We should migrate the model instead, as we do for straight OD models (in _model.py)

Reply:
This is a good catch